index.js

```diff
@@ -44,10 +48,24 @@ const app = new App({
           platform: 'ios',
         },
       },
+      HW_CHANNEL: isDebug ? 'ios-debug' : 'ios',
     },
   },
 });
-const appReady = app.start();
+
+const appReady = new Promise(resolve => {
+  (async () => {
+    try {
+      if (NativeModules.WindowCrypto.isAvailable) {
+        await seedRandom();
+      }
+    } catch (e) {
+      // no random, no problem
```

Comment:
When can we hit that code path again? On older iOS versions, Human Web Lite was turned off. Was that the reason why we do not need to handle it?

Would be good to make it more explicit, why no error handling is needed. Otherwise, it reads a bit fishy that we apparently use cryptography without properly initializing.

Reply:
I would like to make a proper error handling in separate PR. There are probably a lot of edge cases we should consider.